build-system/tasks/presubmit-checks.js

```diff
@@ -846,6 +853,7 @@ const forbiddenTerms = {
       'test/unit/test-mode.js',
       'test/unit/test-motion.js',
       'test/unit/test-mustache.js',
+      'test/unit/test-mutator.js',
```

Comment:
The idea behind this check is to remove all tests that use a top-level `describe` block. Would it be possible to convert this test to use `describes` instead?

Reply:
given test-mutator.js is 1000+ lines, might deserve a separate PR.